### PR TITLE
feature/subject-stepper-implementation-2

### PR DESCRIPTION
**Pull request by FryDay969**

### CODE REVIEW

**Review by Tolik170 — APPROVED** [review]

File: tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx [hunk 1]
```diff
@@ -28,61 +55,85 @@ describe('SubjectsStep test with some data', () => {
...
-    fireEvent.click(secondField)
-    fireEvent.change(secondField, { target: { value: 'English' } })
-    fireEvent.keyDown(secondField, { key: 'ArrowDown' })
-    fireEvent.keyDown(secondField, { key: 'Enter' })
+    fireEvent.click(await secondField)
+    fireEvent.change(await secondField, {
+      target: { value: 'Subject 2' }
+    })
+    fireEvent.keyDown(await secondField, { key: 'ArrowDown' })
+    fireEvent.keyDown(await secondField, { key: 'Enter' })
```
Comment (Yurenko) [inline tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx:124]: I guess that await doesn't need here.

File: tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx [hunk 2]
```diff
@@ -116,18 +167,20 @@ describe('SubjectsStep test with some data', () => {
...
-    fireEvent.click(secondField)
-    fireEvent.change(secondField, { target: { value: 'Ukranian' } })
-    fireEvent.keyDown(secondField, { key: 'ArrowDown' })
-    fireEvent.keyDown(secondField, { key: 'Enter' })
+    fireEvent.click(await secondField)
+    fireEvent.change(await secondField, {
+      target: { value: 'Subject 2' }
+    })
+    fireEvent.keyDown(await secondField, { key: 'ArrowDown' })
+    fireEvent.keyDown(await secondField, { key: 'Enter' })
```
Comment (Yurenko) [inline tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx:183]: The same thing here.

File: tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx (hunk 1 above)
Comment (FryDay969) [reply tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx:124]: No, it is needed. The test will not work without it.

File: tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx (hunk 2 above)
Comment (FryDay969) [reply tests/unit/containers/mentor-home-page/subjects-step/SubjectsStep.spec.jsx:183]: Also neeeded.

**Review by maarekua — APPROVED** [review]

**Pull request merged**